javascript/sdk/lib/types.ts

```diff
@@ -207,22 +207,23 @@ export interface TestData {
   mappings: string[];
 }
 
-export interface Unit {
-  id: string;
-  label: string;
-  definition: string;
-  description: string;
-}
-
-export interface UnitUsage {
-  tests: string[];
+export interface AdvisoryUsage {
   count: number;
   failed: number;
 }
 
-export interface UnitInfo {
-  unit: Unit;
-  usage?: UnitUsage;
+export interface AdvisoryInfo {
+  id: string;
+  usage: AdvisoryUsage;
+}
+
+export interface Advisory {
+  account: string;
```

Comment:
do these fields need to match the json response exactly? (i forget how js works)

if so, it'll be `account_id`